.github/workflows/check-tutorials.js.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 The Google Earth Engine Community Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This workflow will do a clean install of node and run tests.
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
```

Comment:
Nit: The name field here (`Node.js CI`) and the filename (`check-tutorials.js.yml`) are quite different, whereas in the event that other GitHub Actions are added someday we might want them to match. I'd probably omit the `.js` infix as well.

My suggestion would be just "CI / ci.yml" if we expect to add future tests (on things other than tutorials) to this action, or "Check Tutorials / check-tutorials.yml" if not.

Reply:
Iiuc, the `paths` setting is defined at the level of the workflow file. Other checks are currently being added for Landcover Toolkit, which will have a different `paths` value. In that case, perhaps it makes sense to give this workflow a more specific name like "Check Tutorials"?

Reply:
SGTM, whichever you prefer – continuing thread in https://github.com/google/earthengine-community/pull/364/files#r655717853.